Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -779,10 +779,26 @@ static void loadPrimitive(
   }
 
   int materialID = primitive.material;
+
   const Material& material =
       materialID >= 0 && materialID < model.materials.size()
           ? model.materials[materialID]
           : defaultMaterial;
+
+  bool isUnlit = material.hasExtension<ExtensionKhrMaterialsUnlit>() &&
+                 !options.pMeshOptions->pNodeOptions->pModelOptions
+                      ->ignoreKhrMaterialsUnlit;
+  if (isUnlit) {
+    Material& unlitMaterial = const_cast<Material&>(material);
```

Comment:
Can we avoid modifying the material? It's pretty dodgy for a function to take a const parameter, then cast away that const and modify it anyway.